src/communication/azext_communication/manual/_params.py

```diff
@@ -8,17 +8,147 @@ def load_arguments(self, _):
     with self.argument_context('communication update') as c:
         c.argument('location', validator=None)
 
+    _load_identity_arguments(self)
+    _load_sms_arguments(self)
+    _load_phonenumber_arguments(self)
+    _load_chat_arguments(self)
+
+
+def _load_identity_arguments(self):
+    self.argument_context('communication identity create-user')
+
+    with self.argument_context('communication identity delete-user') as c:
+        c.argument('user_id', options_list=['--user-id', '-u'], type=str, help='ACS identifier')
+
     with self.argument_context('communication identity issue-access-token') as c:
-        c.argument('userid', options_list=['--userid', '-u'], type=str, help='ACS identifier')
-        c.argument('scopes', options_list=[
-                   '--scope', '-s'], nargs='+', help='list of scopes for an access token ex: chat/voip')
+        c.argument('user_id', options_list=['--user-id', '-u'], type=str, help='ACS identifier')
```

Comment:
It's a breaking change to change option names from `'--userid', '-u'` to `'--user-id', '-u'`

Reply:
Rolled back this renaming.